.git-blame-ignore-revs

```diff
@@ -0,0 +1,50 @@
+# Pinking shears
+a4789b3cc1cb59e5cd062670f4439f264c0d34f3 # [12042]
+fb4e38b0e750c0cab5eb4c50532da93e2d9882b3 # [15843]
+4b33a0e9c45d67c5f3123577262f73727444254a # [16438]
+fa33255b567df6b19ccc3a61824ad243db116993 # [16439]
+46d96c7704b978b9d070bfec1cb1a33e4caf46b0 # [18254]
+feaf2597bf16b1bd6b489a0e5680501f12990bee # [18276]
+9e7890c3f61781200b9337186d85cd4034edac61 # [18386]
+fd10e500e47ca1ae3280ac6817e725965f7fc0dc # [19054]
+6610e321e79d9b56c6fc496b6630dd2ac664a0d3 # [19528]
+d26f0a4c407a5d13a095c7676b397e51a959d82c # [19577]
+89e9bcc1d69ebbac01c8c4501f9066e421125e66 # [20000]
+8c50f982ea9784fe06844ce84b1a3297b1241442 # [20715]
+b6e23d7269e50296270f93dda96dacb118bee8b3 # [20944]
+48a3ec24c5ea7615431bde431098546674a0096c # [21070]
+2439e4d722f5093631c4bac9a45106ab33624aaa # [21381]
+56c4577feae624f7a3bf55d001295e0707c972b6 # [21486]
+680b671330327f2568549dec9460a2e942219d55 # [21492]
+dd39a3b3d2d1f792b6ed8e88b1fcc13877fbd8dd # [22491]
+6d8bce688f5215ad20133b2cefedc72ac5136be3 # [22634]
+05c0f14024a2bb8ef9bdfcb84589d04205f8ce24 # [23679]
+ece7a7714477142d4cdb3269d2f9ad3d5dab2e5f # [23780]
+687d1a2ce992a0e7e24675b506b85a57e42fe78e # [24303]
+45d2a20783460667f85d5fdced422efd54c33b72 # [24603]
+c8889d984fd98381d6f0bd3228972e93348f1266 # [25085]
+21da24227f2c0087c95f44c60b1bca65cedf0611 # [25824]
```

Comment:
Along with whitespace, also removes a `.` 

I hate me too sometimes...

Reply:
Sigh. Should probably exclude this commit then. What do you think of leaving it in here so if in the future people auditing this for some ghastly reason aren't left wondering if this was missed originally?

```suggestion
# 21da24227f2c0087c95f44c60b1bca65cedf0611 # [25824] includes a punctuation change
```

Reply:
> What do you think of leaving it in here so if in the future people auditing this for some ghastly reason aren't left wondering if this was missed originally?

Works for me, I've committed the suggestion.